javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,25 @@
+//Flight booking fullname function
+
+function getFullName(firstname, surname) {
```

Comment:
You declared this function twice. Here and at line 18. Delete this one. the other one is enough